addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/BackupFileDisclosureScanRule.java

```diff
@@ -723,6 +724,9 @@ private void findBackupFile(HttpMessage originalMessage) throws Exception {
                 sendAndReceive(requestmsg, false);
                 disclosedData = requestmsg.getResponseBody().getBytes();
                 int requestStatusCode = requestmsg.getResponseHeader().getStatusCode();
+                if (isIgnorableMessage(requestmsg)) {
+                    continue;
```

Comment:
I'm sure this is logically correct but it reads awkwardly. If it can be ignored then continue.

You should remove the nots (!) from the function and break if it's ignorable?

Wouldn't that seem more like ignoring what's ignorable.

Reply:
I think that would change the way it works and actually stop it working correctly. We dont want to break if the message is ignorable because the time round it might not be.
Happy to change the name, but not really sure what to :/

Reply:
have changed to isWithinThreshold :)